bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -63,9 +123,26 @@ data:
         export REDIS_REPLICATION_MODE="master"
     else
         export REDIS_REPLICATION_MODE="slave"
+
+        retry_count=1
+        while validate_quorum
+        do
+            info "sleeping, waiting for redis master to come up"
+            sleep 1s
+            if ! ((retry_count % 11)); then
+                info "Trying to manually failover"
+                failover_result=$(trigger_manual_failover)
+                
+                debug "Failover result: $failover_result"
+            fi
+            
+            ((retry_count+=1))
+        done
+        info "redis master is up now"
```

Comment:
Please, modify redis to Redis for the logs.

Reply:
Fixed: https://github.com/bitnami/charts/pull/7461/commits/11a7111c6147ed689babc3bfe4db2a922bacf91b